plugins/rest/unit_test/test_patch.py

```diff
@@ -70,3 +73,44 @@ def test_patch_unit(self):
         self.assertEqual(results["body_object"], {"SampleSuccessBody": "SampleVal"})
         self.assertEqual(results["body_string"], "SAMPLETEXT for method PATCH")
         self.assertEqual(results["headers"], {"SampleHeader": "SampleVal"})
+
+    @parameterized.expand(
+        [
+            ("https://www.google.com", {}, [{"action": "jumps"}, {"over": "dog"}, {"ip": "192.168.0.1"}], {}),
+            ("https://www.google.com", {}, [], {"client_id": "name", "client_secret": "passwd"}),
+        ]
+    )
+    def test_post_with_either_value(self, route, headers, body_as_an_array, body):
+        test_conn = MockConnection()
+        test_action = Patch()
```

Comment:
the name of the test is `test_post_with_either_value` yet you create a `Patch` object